### ISSUE TITLE
More Solar2D rebranding for the installers (e.g., aliases for executable names on Windows/MacOS)

### ISSUE THREAD

**Issue by mjocean** [coronalabs/corona#124]
**Issue:** The "Solar2D" installers install files using the Corona name and results in no folders or executables named Solar2D installed on the system.  I recommend at least installing shortcuts/aliases so that brand new users can find the installed product by searching for the name by which the product currently identifies itself.

I have taught with CoronaSDK for many years, and have had great success with it.  I would like to keep using it, however we all know "Corona" has become a very loaded word.  I understand that renaming the executables installed by the installer will break Sublime Text and other IDE integrations, but without any folders or executables with the correct new name, there isn't even a path by which these dependencies can eventually be resolved.

Thanks!

**Comment by Shchvova** [comment]
It's a very good point. Unfortunatelly many things are dependant on "Corona" naming. But I guess it has to change sooner or later.

**Comment by OutlawGameTools** [comment]
You don’t want to break third-party things just willy-nilly, but if that’s the only reason Corona hasn’t been renamed, then let’s rename it! Those third-party things can be updated. I’ve been putting off doing new tutorial videos partly due to showing how awesome Solar2D is, but doing it with something called Corona SDK. It’s just one more level of “what?!“ that a newbie doesn’t need.

**Comment by mjocean** [comment]
I doubt I'm the only person who is rethinking Corona for the Fall because of the unfortunate name --a name which might trigger who knows what for a student.  

I personally wouldn't mind running a branch in which the renaming is more aggressive but interoperability with other tools may be poor.

**Comment by nikhilCad** [comment]
Any updates?

**Comment by bauerbach** [comment]
I agree. I am actually reluctant to download it because of the "Corona" name!

**Comment by develephant** [comment]
Branding is generally the last thing a developer thinks about, but is highly important for recognition. I'm extremely happy I can use Solar2d but it's hard to recommend because of this issues. I imagine it would be difficult to do.

**Comment by sekodev** [comment]
I think it's time to rebrand the installers, change logos and graphics to Solar2D and move the documentation to docs.solar2d.com. I know moving the documentation might break a thing or two but that could be fixed over time.
